rfcs/0093-rfc-categories.md

```diff
@@ -0,0 +1,152 @@
+---
+feature: rfc_categories
+start-date: 2021-05-19
+author: David Arnold
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @kevnicox, @Mic92, @gytis-ivaskevicius
+shepherd-leader: @kevincox
+related-issues:
+category: process
+---
+
+# Summary
+[summary]: #summary
+
+The RFC process is amended with means to categorize RFCs into one of: _feature_, 
+_information_ or _process_, where each category sets different accents that
+improve the overall outcome of the process.
+
+# Motivation
+[motivation]: #motivation
+
+## Some issues are not addressed by the community
+
+There is no appropriate venue for it. When reading the current RFC process, it becomes
+clear that it was made primarily with purely technical decisions in mind. It might only
+be a perception, but this perception systemically skews what's being discussed and what
+not.
```

Comment:
But then why are there existing RFCs which you have yourself marked as `informational` or `process`? This seems to contradict your point.

Reply:
I think it is the best exemplification of the need for this RFC, indeed.

It seems so necesary that people even disregarded the skewed conductive formulation of the current RFC material.

On the other hand, not everyone might feel entitled to this "disregard".

Hence, this RFC intended to empower authors of  RFCs that are non-feature in order to help scale the governance process.